packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -99,13 +129,22 @@ class RowLarge extends React.Component {
 				aria-label={titleField && getCellData(titleField, parent, index)}
 			>
 				<div className={`tc-list-large-inner-box ${theme['inner-box']}`} key="inner-box">
-					<div className={theme.header} key="header">
-						{titleCell}
-						{selectionCell}
-					</div>
-					<dl className={`tc-list-large-content ${theme.content}`} key="content">
-						{otherFields.map(this.renderKeyValue)}
-					</dl>
+					{isEmpty(rowData) ? (
+						<LargeInnerRowLoading
```

Comment:
RV pass a prop `isScrolling` so everytime you scroll, the row is rendered. You use a random to get the size of each skeleton. The combination of that is that on scroll, the size of the skeletons change.
To avoid that, you can use React.memo:
```diff
+const MemoLargeLoading = React.memo(LargeInnerRowLoading);

...
-<LargeInnerRowLoading />
+<MemoLargeLoading />
```